src/contracts/FoxStaking.sol

```diff
@@ -259,12 +262,27 @@ contract FoxStaking is Ownable {
     }
 
     struct Claim {
-        uint256 deposit;
+        uint256 amount;
         uint256 gons;
         uint256 expiry;
         bool lock; // prevents malicious delays
     }
     mapping(address => Claim) public warmupInfo;
+    mapping(address => Claim) public cooldownInfo;
+
+    /**
+        @notice creates a withdrawRequest with Tokemak
+        @param _amount uint
+        @return bool
+     */
+    function requestWithdrawalFromTokemak(uint256 _amount)
+        internal
+        returns (bool)
+    {
+        ITokePool tokePoolContract = ITokePool(tokePool);
+        tokePoolContract.requestWithdrawal(_amount);
+        return true; // TODO: TOKE requestWithdrawal function doesn't return anything.  Need to check for proper event emitted
```

Comment:
Any reason to return true here? It doesn't appear that we are using the return value.

Reply:
No, I was going with the format of stake, I wanted to talk to you about that.  I am concerned that requestWithdrawal will silently fail, but I'll leave the comment to make sure I investigate that later.